README.md

```diff
@@ -42,6 +42,7 @@
 
 - `Mac`:
     Start the terminal program (Search for `Terminal`). Install the `Command Line Tools` using the command `xcode-select --install`. This will also install `git`. Alternatively, you can install Xcode and install the `Command Line Tools` after you run it the first time.
+    It is also recommended to install brew (a package manager) with the following command: `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
```

Comment:
@Parker-Corbitt why is it necessary to install brew? It's probably a good idea to have it, but it's not clear why it's needed now.

Reply:
I suppose it isn't necessary to install right at the start. My Mac knowledge is limited, but I believe aptitude doesn't come pre installed any longer. I figured it would be easiest to just go ahead and add brew to the list of items necessary to install, since every other environment has a package manager already set up.

Reply:
Got it